tests/phpunit/tests/post/wpCountAttachments.php

```diff
@@ -28,6 +28,7 @@ public function test_wp_count_attachments_should_cache_the_result() {
 		$expected = wp_count_attachments( $mime_type );
 		$actual   = wp_cache_get( $cache_key, 'counts' );
 
-		$this->assertEquals( $expected, $actual );
+		$this->assertSame( $expected->$mime_type, $actual->$mime_type );
+		$this->assertSame( $expected->trash, $actual->trash );
```

Comment:
While I agree that using `assertSame` is better here, the assertions replacing the original assertions are not equivalent to what the original assertion checked (what if a new property gets added or another one removed ?). The assertions are also unstable as no check is done on whether the property _exists_ on the `$actual` object, but rely on the assumption that it will.

Reply:
Hi @jrfnl,

Thank you for the review.

I updated it to check these two properties because `wp_count_attachments` only adds them.

If a property gets removed, I'd say the test should fail because it would break codes that rely on the removed property. If a new property gets added, the tests should be updated to check that.

It seems that a new test should be created to check the expected properties. Does it make sense?